leshan-client-core/src/main/java/org/eclipse/leshan/client/datacollector/DataSenderManager.java

```diff
@@ -0,0 +1,131 @@
+package org.eclipse.leshan.client.datacollector;
+
+import org.eclipse.leshan.client.request.LwM2mRequestSender;
+import org.eclipse.leshan.client.resource.LwM2mRootEnabler;
+import org.eclipse.leshan.client.send.NoDataException;
+import org.eclipse.leshan.client.servers.ServerIdentity;
+import org.eclipse.leshan.core.Destroyable;
+import org.eclipse.leshan.core.Startable;
+import org.eclipse.leshan.core.Stoppable;
+import org.eclipse.leshan.core.node.LwM2mNode;
+import org.eclipse.leshan.core.node.LwM2mPath;
+import org.eclipse.leshan.core.node.TimestampedLwM2mNodes;
+import org.eclipse.leshan.core.request.ContentFormat;
+import org.eclipse.leshan.core.request.ReadCompositeRequest;
+import org.eclipse.leshan.core.request.SendRequest;
+import org.eclipse.leshan.core.response.ErrorCallback;
+import org.eclipse.leshan.core.response.ReadCompositeResponse;
+import org.eclipse.leshan.core.response.ResponseCallback;
+import org.eclipse.leshan.core.response.SendResponse;
+
+import java.util.*;
```

Comment:
There is some organize import issue but I hope with :
- https://github.com/eclipse/leshan/issues/1265#issuecomment-1152194599
- https://github.com/eclipse/leshan/issues/1265#issuecomment-1152551302

I should provide tools to help contributor to fix this kind of issue.

By the way, which IDE are you using ?

Reply:
I'm using IntelliJ for development, but I imported the project into Eclipse for formatting. I tried to organize imports, but I'm getting a message: _The resource is not on the build path of a Java project_. It seems to be an issue of a Maven Build run configuration, but I can't add a new Java Application configuration for some reason.

Reply:
That's strange.

Did you try : https://github.com/eclipse/leshan/wiki/Code-&-design-guidelines#eclipse-workspace-setup ?

Anyway, I hope the new maven way will simplify IntelliJ user. (not yet ready)

Reply:
IntelliJ loves to bunch up import classes using `*` if there are more than 3 or 4 from the same package. I guess i will have to look out for that and change it manually if that ever happens again. Anyway, I fixed it now.